doc/api.md

```diff
@@ -134,6 +135,32 @@ E.g.:
     ]
 }
 ```
+OR
```

Comment:
I'll suggest swapping the order of the examples, as I think the one should appear first is "configuration groups", as "service" is a kind of legacy that we want to deprecate

Reply:
Fixed in 424d59d